### DISCUSSION TITLE
Shadowlands 9.2.7 non-official TC! Crash

### DISCUSSION THREAD

**Discussion by Acid2020** [TrinityCore/TrinityCore#29248] [General]
hello, I downloaded this assembly https://github.com/freadblangks/AzgathCoreSL_F   pulled out maps with an extractor from 9.0.2, since the extractor does not work with 9.2.7, but worldserver gives an error and closes! Who understands, tell me, this error is in the core or the client is not suitable!

ERROR!!!    src\server\game\DataStores\DB2Stores.cpp:544 in LoadDB2 ASSERTION FAILED throws:
  clientMetaString == ourMetaString JournalEncounter.db2 C++ structure fields issffhhihhibb do not match generated types from the client ssffihhihhibb        Never encountered this error before

**Comment by Aokromes** [comment]
ask supporpt on that fork.
